src/test/scala/edu/ie3/util/quantities/QuantityUtilSpec.scala

```diff
@@ -186,4 +187,28 @@ class QuantityUtilSpec extends UnitSpec with TableDrivenPropertyChecks {
       }
     }
   }
+
+  "Converting units to alternative units" should {
+    "succeed if units are compatible" in {
+      val cases = Table(
+        ("sourceUnit", "targetUnit", "expectedUnit"),
+        (VOLTAMPERE, WATT, WATT),
+        (KILOVOLTAMPERE, WATT, KILOWATT),
+        (MEGAVOLTAMPERE, WATT, MEGAWATT),
+        (VAR, WATT, WATT),
+        (KILOVAR, WATT, KILOWATT),
+        (MEGAVAR, WATT, MEGAWATT),
+        (VOLTAMPERE, VAR, VAR),
+        (KILOVOLTAMPERE, VAR, KILOVAR),
+        (MEGAVOLTAMPERE, VAR, MEGAVAR),
+        (WATT, VAR, VAR),
+        (KILOWATT, VAR, KILOVAR),
+        (MEGAWATT, VAR, MEGAVAR)
```

Comment:
```suggestion
        (MEGAWATT, VAR, MEGAVAR),
        (VAR, VOLTAMPERE, VOLTAMPERE),
        (KILOVAR, VOLTAMPERE, KILOVOLTAMPERE),
        (MEGAVAR, VOLTAMPERE, MEGAVOLTAMPERE),
        (WATT, VOLTAMPERE, VOLTAMPERE),
        (KILOWATT, VOLTAMPERE, KILOVOLTAMPERE),
        (MEGAWATT, VOLTAMPERE, MEGAVOLTAMPERE)
```
Should we add also the remaining two cases, then the test covers every possibility?